lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -49,6 +49,15 @@ def self.edit_changelog(prepopulated_changelog, changelog_latest_path, editor)
         end
       end
 
+      def self.write_changelog(prepopulated_changelog, changelog_latest_path)
+        UI.user_error!("Pre populated content for changelog was empty") if prepopulated_changelog.empty?
+
+        UI.message("Using pre populated contents:\n#{prepopulated_changelog}")
```

Comment:
Hmm I wondered if this would be too long to log, but considering this would be automated, I think that this is useful 👍

Reply:
I think it's useful yes